target/linux/ramips/image/mt7621.mk

```diff
@@ -83,6 +83,17 @@ define Build/ubnt-erx-factory-image
 	fi
 endef
 
+define Build/yuncore-ax820-metadata
+	echo \
+		'{ \
+			"device_code": [".*"], \
+			"hwver": [".*"], \
+			"batch": [".*"], \
+			"serial": [".*"], \
+			"supported_devices":["mt7621-rfb-ax-nor"] \
+		}' | fwtool -I - $@
+endef
```

Comment:
What's supposed to do? Hacking in metadata is a NAK from me.

Reply:
ACK, I wasn't sure if this makes sense, removed and updated flash instructions.

The factory firmware detects the device as `mt7621-rfb-ax-nor` but uses the sysupgrade metadata format otherwise, with this the factory.bin could be directly flashed.
There are quite a few devices/vendors that do this, is there any proper way to build factory-specific sysupgrade images?

Reply:
Have you tried adding
SUPPORTED_DEVICES += mt7621-rfb-ax-nor
?

Reply:
I'm leaving this as-is, I don't think its useful to have a factory firmware for this device, since the other YunCore devices don't have one either.